### ISSUE TITLE
Release artifacts are excessively large

### ISSUE THREAD

**Issue by holly-hacker** [githubnext/monaspace#69]
I just downloaded the release zip file of Monaspace 1.000 and was surprised that it was over 100MB in size. This seems to be because it contains a copy of the entire repo, which includes 92MB of image files in `docs/images`. Including this in the release distribution is pretty pointless and massively bloats the download size. It's also pointless, because these images are already visible in the README and on the website. Also, each github release already has an option to download a snapshot of the repo at the time of release so this option is redundant.

Please change the release process to either include a zip file with just the fonts, or better yet zip each font type (ie. `otf`, `variable` and `webfonts`) separately. This would cut down the file sizes down to ~25MB or 11MB/1MB/12MB respectively.

In the western world bandwidth may seem ubiquitous, but that is not the case everywhere. This is a massive waste of resources and I am disappointed that GitHub of all companies did not keep this in mind.

**Comment by 33b5e5** [comment]
Also whoever deployed the release left in the `.DS_Store` and `__MACOSX` files/folders, which admittedly don't take up much space, but are completely unnecessary (even on macOS, but especially on Linux and Windows).

**Comment by kkarhan** [comment]
Agreed, the [release zip does contain bloat](https://github.com/githubnext/monaspace/issues/69#issuecomment-1807444062) that should be excluded via the ```.gitignore``` file.

**Comment by holly-hacker** [comment]
I'm not sure if this is meant to be funny, but this got closed as v1.200 got released whose single release artifact is **948mb** in size. That is almost a gigabyte. It follows none of the advice in this issue, such as:
- Packaging different fonts separately so people don't need to download every font format of every single font family in every single weight (even if just variable width ttf may have sufficed).
- Offering alternative compression formats which massively reduce archive size
- Removing the 92mb of image assets that only serve as documentation, which includes a bunch of gif files which are already terribly inefficient.

Additionally, the release zip now contains:
- 4 different font formats (woff, woff2, ttf and otf)
- ... for 5 different fonts
- ... each with 42 different font weights (excluding variable fonts)

Surely you don't expect everybody to use all 855 of these files.

@idan @heathercran This issue was closed after the release was made. Since both of you work for GitHub, surely you must know that you can add/remove release artifacts after the release itself was made? What was the thought process behind closing this issue? Is this simply not planned? Any kind of official communication would be appreciated.

**Comment by idan** [comment]
I'll look into this for 1.3

**Comment by holly-hacker** [comment]
@idan Any chance you'll be able to retroactively split up the release artifact for the v1.200 release? A 1gb download is quite excessive when users may just want 1 font format or 1 font.

**Comment by idan** [comment]
Solved in [1.3](https://github.com/githubnext/monaspace/releases/tag/v1.300)!
